lib/Cmts.ml

```diff
@@ -280,6 +280,24 @@ let relocate (t : t) ~src ~before ~after =
           r |> Location.Set.remove src |> Location.Set.add after
           |> Location.Set.add before) )
 
+let relocate_match_cmts (t : t) src tok ~whole_loc ~matched_loc =
+  let f map =
+    let f Cmt.{loc= cmt_loc; _} =
+      Source.is_before_match_keyword src tok ~exp_loc:whole_loc ~cmt_loc
+    in
+    let before, after =
+      List.partition_tf (Location.Multimap.find_multi map matched_loc) ~f
+    in
+    let map =
+      List.fold_left ~init:map (List.rev before) ~f:(fun map ->
+          Location.Multimap.add_multi map whole_loc)
+    in
+    Location.Multimap.change_multi map matched_loc after
+  in
+  update_cmts t `Before ~f ;
+  update_cmts t `After ~f ;
```

Comment:
Why do we call `update_cmts` with `After` ?

Reply:
it's unnecessary indeed